src/commands/user/export-mapping.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+
+import { flags } from '@oclif/command'
+import { Kommand } from '../../common'
+import { kuzzleFlags } from '../../support/kuzzle'
+
+export default class UserExportMapping extends Kommand {
+  static description = 'Exports user mapping to JSON.'
+
+  static flags = {
+    help: flags.help({}),
+    path: flags.string({
+      description: 'Dump directory',
+      default: 'userMapping'
+    }),
+    ...kuzzleFlags,
+    protocol: flags.string({
+      description: 'Kuzzle protocol (http or websocket)',
+      default: 'ws'
+    })
+  }
+
+  async runSafe() {
+    const filename = path.join(this.flags.path, 'userMapping.json')
```

Comment:
Mappings should be in the plural form, also we are talking about the `users` collection
```suggestion
    const filename = path.join(this.flags.path, 'users-mappings.json')
```